services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -419,4 +419,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         <addAutoIncrement columnDataType="BIGINT" columnName="CHALLENGE_MANAGER_ID" incrementBy="1" startWith="1"
                           tableName="CHALLENGE_MANAGER_RULE"/>
     </changeSet>
+    <changeSet id="1.0.0-37" author="exo-gamification" dbms="mysql,mssql,h2,sybase,db2,hsqldb,postgresql">
```

Comment:
this changeset is specific to Mysql, do you reproduce the problem on Postgresql ?
Otherwise, we do support just Mysql and Postgresql, no need to mention other RDBMS

Reply:
roger that !
i'll verify Postgresql's influence on the problem 
I want to mention that locally the problem is not reproduced, it's reproduced on the acceptance .
the instance logs of acceptance are pointing out that it's an SQL column type issue 'COMMENT' column .